acceptance-tests/src/test/resources/features/site/facets.feature

```diff
@@ -136,6 +136,6 @@ Feature: Faceted search
         Given I navigate to the "search" page
         Then I should see the list with title "PUBLICATION STATUS" containing:
             | Published ( ... |
-            | Upcoming (2) |
+            | Upcoming (3) |
```

Comment:
this might break on travis :D

Reply:
you need to clean-up after the "scheduled publication date format" test, to make test.cms + test.site pass :)

Reply:
good spot.